src/routes/routeConstants.ts

```diff
@@ -13,43 +18,188 @@ export const COLONY_EXTENSIONS_ROUTE = `extensions`;
 export const COLONY_INTEGRATIONS_ROUTE = `integrations`;
 export const COLONY_INCORPORATION_ROUTE = `incorporation`;
 export const COLONY_ADVANCED_ROUTE = `advanced`;
-export const COLONY_EXTENSION_DETAILS_ROUTE = `extensions/:extensionId/*`;
-export const COLONY_EXTENSION_DETAILS_SETUP_ROUTE = `extensions/:extensionId/setup`;
-export const COLONY_FUNDING_ROUTE = `funds`;
+export const COLONY_EXTENSION_DETAILS_ROUTE = `${COLONY_EXTENSIONS_ROUTE}/:extensionId/*`;
+export const COLONY_EXTENSION_SETUP_ROUTE = `setup`;
+export const COLONY_EXTENSION_DETAILS_SETUP_ROUTE = `${COLONY_EXTENSIONS_ROUTE}/:extensionId/${COLONY_EXTENSION_SETUP_ROUTE}`;
+export const COLONY_BALANCE_ROUTE = `balance`;
 export const COLONY_MEMBERS_ROUTE = `members`;
 export const COLONY_MEMBERS_WITH_DOMAIN_ROUTE = `members/:domainId`;
 export const COLONY_CONTRIBUTORS_ROUTE = `contributors`;
 export const COLONY_FOLLOWERS_ROUTE = `followers`;
 export const COLONY_VERIFIED_ROUTE = `verified`;
-export const COLONY_BALANCE_ROUTE = `balance`;
 export const COLONY_TEAMS_ROUTE = `teams`;
+export const COLONY_AGREEMENTS_ROUTE = `agreements`;
+export const COLONY_DECISIONS_ROUTE = `decisions`;
 export const COLONY_DECISIONS_PREVIEW_ROUTE = `decisions/preview`;
 export const DECISIONS_PAGE_ROUTE = `decisions/tx/:transactionHash`;
```

Comment:
These can be removed, right?

Reply:
Yes but useful to be able to access old decisions ui until agreements ui has been completed, no?